PWGHF/Utils/utilsEvSelHf.h

```diff
@@ -297,6 +344,28 @@ struct HfEventSelection : o2::framework::ConfigurableGroup {
           SETBIT(rejectionMask, EventRejection::Occupancy);
         }
       }
+
+      if (useUpcTrigger) {
+        if (!bcs) {
+          throw std::runtime_error("UPC selection requested but no BC table provided!");
+        }
+        SGCutParHolder sgCuts = setSGPreselection();
+        auto bc = collision.template foundBC_as<BCs>();
+        auto bcRange = udhelpers::compatibleBCs(collision, sgCuts.NDtcoll(), *bcs, sgCuts.minNBCs());
+        auto isSGEvent = sgSelector.IsSelected(sgCuts, collision, bcRange, bc);
+        int issgevent = isSGEvent.value;
```

Comment:
Please do not define variables whose names just differ in capitalisation.

Reply:
fixed